src/Inspection.php

```diff
@@ -27,30 +26,35 @@ class Inspection
      * @throws ConfigurationException
      * @throws InspectionsProfileException
      */
-    public function __construct(string $projectPath, string $inspectionsXmlPath, bool $verbose)
+    public function __construct(Configuration $configuration)
     {
-        $projectDirectory = new ProjectDirectory($projectPath);
+        $projectDirectory = new ProjectDirectory($configuration->getProjectDirectory()->getPath());
```

Comment:
Doesn't need to exist at this level, can just pass a Directory to the inspection command